### PR TITLE
Made autocompletion more robust, by rendering maximum 5 usernames.

### PR DESCRIPTION
**Pull request by mohitanand001**

### CODE REVIEW

**Comment by souravbadami** [comment]
@farziengineer -- Could you fix the merge conflicts ?

**Comment by farziengineer** [comment]
@souravbadami Did it.

File: website/static/js/jquery.caret.js [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+$.fn.caret.EditableCaret = EditableCaret;
+
+$.fn.caret.InputCaret = InputCaret;
+
+$.fn.caret.Utils = Utils;
+
+$.fn.caret.apis = methods;
+
+
+}));
```
Comment (souravbadami) [inline website/static/js/jquery.caret.js]: How about using this over a CDN ? It will make things faster.

File: bugheist/urls.py [hunk 2]
```diff
@@ -19,45 +19,41 @@
...
+    url(r'^issue/comment/add/$',comments.views.add_comment, name='add_comment'),
+    url(r'^issue/comment/delete/$',comments.views.delete_comment, name='delete_comment'),
+    url(r'^comment/autocomplete/$',comments.views.autocomplete, name='autocomplete'),
+    url(r'^issue/comment/(?P<pk>\d+)/edit/$',comments.views.EditCommentPage, name='edit_comment'),
+    url(r'^issue/comment/(?P<pk>\d+)/update/$',comments.views.EditComment, name='update_comment'),
+    url(r'^social/$', TemplateView.as_view(template_name="social.html")),
+    url(r'^search/$', website.views.search),
+    url(r'^report/$', TemplateView.as_view(template_name="report.html")),
+    url(r'^i18n/', include('django.conf.urls.i18n')),
+   ) + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```
Comment (souravbadami) [inline bugheist/urls.py]: Please follow pep8 in this and all files in this PR. It'll be causing linter failure in the future.

**Review by souravbadami — CHANGES_REQUESTED** [review]

**Comment by souravbadami** [comment]
Hi @farziengineer -- I've added few comments. It looks awesome though!

File: website/static/js/jquery.caret.js (hunk 1 above)
Comment (farziengineer) [inline website/static/js/jquery.caret.js]: I tried the cdn thing, but it was causing some mixed content error, something along the line of mixing of http and https

File: bugheist/urls.py (hunk 2 above)
Comment (farziengineer) [inline bugheist/urls.py]: pep8 has many conventions, Which all should be followed?

**Pull request closed without merging**